pkg/action/install.go

```diff
@@ -112,7 +113,9 @@ type Install struct {
 	UseReleaseName bool
 	// TakeOwnership will ignore the check for helm annotations and take ownership of the resources.
 	TakeOwnership bool
-	PostRenderer  postrender.PostRenderer
+	// Also considers the state of custom resources and custom resource definitions in the cluster when adopting resources.
+	ThreeWayMergeForCustomResources bool
```

Comment:
Can we add a `TODO Helm 4: Remove ..` here please
```suggestion
	// Also considers the state of custom resources and custom resource definitions in the cluster when adopting resources.
	// TODO Helm 4: Remove this config and always merge custom resources
```

For Helm 4, we should always look to three-way merge CRs IMHO.

(and same below for upgrade/rollback below)